MiguelAValdesG/SpringDataJPA/db_versions/db_0.1.0/db_1.sql

```diff
@@ -0,0 +1,44 @@
+
+CREATE DATABASE IF NOT EXISTS vetPatitas;
+
+CREATE TABLE IF NOT EXISTS vetPatitas.addresses (
```

Comment:
dunno if there is a convention, but I think naming tables in singular foster readibility in your code... example: it avoids naming methods such as getAddresses() when actually just getting a single address... right???

Reply:
I don´t know if there is a convention to, that is the way in the I learned, but you're right, I understand your point and in fact I'm agree with this, I did this changes. Thank you